apps/api/src/server/applications/application/documents/__tests__/document.service.test.js

```diff
@@ -81,19 +108,21 @@ describe('Document service test', () => {
 	test('obtainURLForDocumentVersion uploads new version of document', async () => {
 		databaseConnector.case.findUnique.mockResolvedValue(application);
 		databaseConnector.document.findUnique.mockResolvedValue(documentWithVersions);
-		got.post.mockReturnValue({
-			json: jest.fn().mockResolvedValue({
-				blobStorageHost: 'blob-store-host',
-				blobStorageContainer: 'blob-store-container',
-				documents: [document]
-			})
-		});
 
 		const ressponse = await obtainURLForDocumentVersion(document, caseId, documentGuid);
 
 		expect(ressponse.blobStorageHost).toEqual('blob-store-host');
-		expect(ressponse.blobStorageContainer).toEqual('blob-store-container');
-		expect(ressponse.documents).toEqual([document]);
+		expect(ressponse.privateBlobContainer).toEqual('blob-store-container');
+		expect(ressponse.documents).toEqual([
```

Comment:
@chrisdalykinandcarta I've had to change these tests, be good to check these. The test before mocked the document-storage-api, but I don't think accurately.